modules/git/repo_language_stats_gogit.go

```diff
@@ -99,7 +107,7 @@ func (repo *Repository) GetLanguageStats(commitID string) (map[string]int64, err
 				if language, has := attrs["linguist-language"]; has && language != "unspecified" && language != "" {
 					// group languages, such as Pug -> HTML; SCSS -> CSS
 					group := enry.GetLanguageGroup(language)
-					if len(group) == 0 {
+					if len(group) != 0 {
```

Comment:
Maybe some parts of this could be backport to v1.15

Reply:
1.15 is fine - this was an error I added in the previous pr.